src/fraction.ts

```diff
@@ -1,4 +1,5 @@
 import BN from "bn.js";
+import {numberToFraction} from "./float_parser";
```

Comment:
Any reason you want to expose this functionality publicly in a separate file? I'd assume people should use Fraction.fromNumber so we could just inline this function privately in this file